index.js

```diff
@@ -9,27 +9,6 @@ const spawn = require('cross-spawn');
 const chalk = require('chalk');
 const which = require('which');
 
-const exitOnError = async function(promise) {
-    try {
-        await promise;
-    } catch (e) {
-        console.log('Error: ', e);
-        process.exit(1);
-    }
-};
-
-const createProject = {
-    command: '$0 <projectDir>',
-    desc: 'create a new blank react project',
-    builder: (yargs) => yargs
-        .option('projectDir', {
```

Comment:
Listing this as an `option` made it show up in the help output alongside `--rust` and `--vanilla` options, which is not correct. I've moved a streamlined version of this `.command` right into the `yargs` block at the bottom.